File: python/29_count_digits.py
"""Contar dígitos — Fácil · Números

Devuelve cuántos dígitos tiene un entero; el signo negativo no cuenta y 0
tiene un dígito.

Fuente: https://coding-challenges.dev/problems/python-contar-digitos
"""


def count_digits(n: int) -> int:
    return len(str(abs(n)))

File: typescript/29-count-digits.ts
// Contar dígitos — Fácil · Números
// El signo negativo no cuenta; el número 0 tiene un dígito.
// Fuente: https://coding-challenges.dev/problems/contar-digitos

function countDigits(n: number): number {
  return Math.abs(Math.trunc(n)).toString().length;
}

// No modificar: necesario para evaluar el resultado.
export { countDigits };
